test/integration/render-tests/distance/paint-polygon/style.json

```diff
@@ -0,0 +1,1480 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 500,
```

Comment:
@zmiao This is quite a large size for a render test. Would it be possible to make a smaller test image?

Reply:
The intention of this test case is to simulate a real world route and to see if it works. That's why the image size is very big, but I will try to decrease the zoom level and minimize the size.

Reply:
Thanks for the context @zmiao. If it's OK to have one larger size image, I do think it would be valuable to have at least one render test that accurately reflects the real world route line use case.